data/org.freedesktop.impl.portal.Settings.xml

```diff
@@ -43,6 +43,13 @@
           Unknown values should be treated as 0 (no preference).
         </para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>org.freedesktop.appearance accent-color (uuu)</term>
+        <listitem><para>
+          Indicates the system's preferred accent color as a tuple of rgb values in the range [0,255].
+          To indicate that a color is not set, use an invalid RGB value, such as (256 256 256).
```

Comment:
Rather than telling implementations what to set, we should probably approach it the other way and define the behavior; e.g.

```suggestion
          Out-of-range RGB values should be treated as an unset accent color.
```